src/Neo/VM/Helper.cs

```diff
@@ -217,6 +217,9 @@ public static ScriptBuilder EmitPush(this ScriptBuilder builder, object obj)
                 case short data:
                     builder.EmitPush(data);
                     break;
+                case char data:
+                    builder.EmitPush((ushort)data);
```

Comment:
we do not allow unicode. so use `byte`

Reply:
ill put it another way, we support char and consider it as ushort. this pr just fix the push method.

Reply:
How you doing to convert a `21-bit` UTF-16 to a `char`? That would require an `int32`. At least use `Convert.ToUInt16` or `(ushort)(object)data`

Reply:
sure，will update and add ut